e2e/cypress/integration/files_and_attachments/image_link_preview_spec.js

```diff
@@ -89,4 +89,76 @@ describe('Image Link Preview', () => {
         // # All image previews expand back open
         cy.findAllByLabelText('file thumbnail').should('be.visible').and('have.length', 4);
     });
+
+    it('MM-T1447 Images below a min-width and min-height are posted in a container that is clickable', () => {
+        const IMAGE_BELOW_MIN_SIZE_1 = 'images-below-min-size-1.png';
+
+        // # Upload Image with min dimensions as attachment and post it
+        cy.get('#fileUploadInput').attachFile(IMAGE_BELOW_MIN_SIZE_1);
+        cy.postMessage(`${Date.now()}-${IMAGE_BELOW_MIN_SIZE_1}`);
```

Comment:
Could move uploading and posting inside `verifyLastAttachedImageHasMinSizeAndOpensPreviewInModal`, so it can be,
```javascript
testCases.forEach((testCase) => {
    verifyLastAttachedImageHasMinSizeAndOpensPreviewInModal(testCase);
});
```